### DISCUSSION TITLE
New Product Gallery is available ?

### DISCUSSION THREAD

**Discussion by Marc-pi** [woocommerce/woocommerce#44943] [WooCommerce, Blocks, and Site Editing]
Hello,
is the new product gallery out or still behind an experimental flag ?
I wanted to edit its block settings, but seems it is not yet available (neither as template part )
it's announced on woocommerce-blocks-11-2-0, but perhaps i missed something ?

**Comment by gigitux** [comment]
Hello, @Marc-pi!
We are still working on the Product Gallery. Currently, it is still behind an experimental flag. 
When it is released, we will mention it in the post, not just in the changelog.

Also, if you want to follow the progress/give us feedback, #9891 is the tracking issue.

**Comment by Marc-pi** [comment]
ok, thanks for the confirmation
hope it will land soon, i need to inactive zoom out
seem the tasks are nearly complete :-) the full view template part is being worked on

**Comment by Marc-pi** [comment]
@gigitux hello, we are building custom grouped product templates (block version / custom plugin), and want to make use of the new product gallery.
we are relativerly new incomers in the woo/wp ecosystem :-)

> Can you explain us how we can technically activate it to use it sooner than the official public inclusion into the woo-blocks plugin ?
 throught the beta tester ?
i checked the woo advanced settings, i cannot find the flag there

> public release of the new gallery block ? what's the target ?

thanks a lot for your help
marc

**Comment by danieldudzic** [comment]
@Marc-pi I appreciate that you are anticipating and hoping to utilize the Product Gallery block!

I'm afraid, I would discourage trying to use it in any capacity other than experimental, at this time.

There are still bugs and a few key things we are working on (for example the thumbnails and the dialog/lightbox), which may introduce breaking changes to the block as it's available (behind the experimental flag) now.

**Comment by Marc-pi** [comment]
yep, we decided yesterday to use the single image instead in between, we found in the doc where the experimental flag was set
from the EPIC https://github.com/woocommerce/woocommerce-blocks/issues/9891, we understood the final shipping is not very far from now ;-)
keep it up.

**Comment by Marc-pi** [comment]
@gigitux / @danieldudzic does the new gallery manage also well pinch to zoom (usefull for mobile) ?
